src/open_inwoner/accounts/templates/registration/login.html

```diff
@@ -52,10 +52,7 @@ <h1 class="utrecht-heading-1">{% trans 'Welkom' %}</h1>
                                     {% endrender_card %}
                                 {% endif %}
                             {% endrender_column %}
-                        {% endif %}
-                        {# end digid #}
 
-                        {% if settings.DIGID_ENABLED %}
```

Comment:
I think that the block below this (OpenID Connect for regular users) should be outside this `DIGID_ENABLED` conditional, since it is a regular OIDC login via Azure or Keycloak (without DigiD)

Reply:
Do you _think_ or are your sure? I also _think_ this... so I can change this if we are sure.

Reply:
I always comment with a small degree of uncertainty, yes I'm sure :stuck_out_tongue: